README.md

```diff
@@ -3,6 +3,30 @@
 
 kafka-connect-venafi-tpp is a [Kafka connector](http://kafka.apache.org/documentation.html#connect) for Venafi Trust Protection Platform security events.
 
+
+Description
+---
+This connector connects via HTTP to your instance of the Venafi Trust Protection Platform ( which shall be refered to TPP from here on) and pulls your Log events into kafka allowing you to do any filtering/transforming/processing you'd like to do within a comfortable kafka environment.  
+ **N.B.** Currently the connector starts from the of the beginning of time, a future release will allow the option of starting from now.
+
+EventLog object produced by this source **Connector** as defined by the Venafi WebSDK
+* **ClientTimestamp**: The time that the client generated the event.
```

Comment:
```suggestion
* **ClientTimestamp**: The time that the client generated the event at.
```
Check with another native speaker. Cf. this article (esp. page 2) https://www.quickanddirtytips.com/education/grammar/ending-a-sentence-with-a-preposition?page=1. Part of me wants a preposition in that sentence, part of me thinks it's redundant, as in "The time the client called me." - maybe `The time at which the client generated the event.`?

Reply:
This is taken pretty much verbatim from the documentation, so...